pkg/gui/view_helpers.go

```diff
@@ -392,3 +392,17 @@ func (gui *Gui) pageDelta(view *gocui.View) int {
 
 	return delta
 }
+
+func getTabbedView(gui *Gui) *gocui.View {
```

Comment:
I added the ParentView field to my gocui fork a while back, before I introduced the concept of Contexts. I haven't officially deprecated it but it is in essence deprecated in favour of Contexts. We have a Context stack that typically starts at the bottom with the current side view and in this case ends at the top with our menu view. The `gui.currentStaticContext()` function starts at the top of the stack and goes down until it finds a static (i.e. non-popup) context.

I reckon we should do the following:
```go
func getTabbedView(gui *Gui) *gocui.View {
	// assumes only static contexts have tabs which is an assumption I consider safe
	context := gui.currentStaticContext()
	view, _ := gui.g.View(context.GetViewName())
	return view
}
```

Reply:
Updated to static contexts, and removed the `ParentView` assignment to not add code using a deprecated field.